test/issue223-1.js

```diff
@@ -0,0 +1,26 @@
+
+var tape = require('tape');
+var mjAPI = require("../lib/mj-single.js");
+var jsdom = require('jsdom').jsdom;
+
+tape('displayAlign:left in HTML output', function(t) {
```

Comment:
Your test says it is about `displayAlign`, but seems to be about `displayIndent` according to the `config()` call below.